ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/datepicker.html

```diff
@@ -28,11 +28,14 @@
      data-cmp-visible="${datePicker.visible ? 'true' : 'false'}"
      data-cmp-enabled="${datePicker.enabled ? 'true' : 'false'}"
      id="${component.id}"
+     data-sly-test.widgetId="${component.id}-datePicker"
```

Comment:
Pls explain why widgetId is required only for datepicker and not for any other component.